src/Objs/Job/JobComponent.js

```diff
@@ -35,19 +41,38 @@ Scrivito.provideComponent("Job", ({ page }) => {
   );
 });
 
-const JobDatePosted = Scrivito.connect(({ datePosted }) => {
+function JobDatePosted({ datePosted }) {
   if (!datePosted) {
     return (
-      <InPlaceEditingPlaceholder block>
-        Select a date in the job page properties.
+      <InPlaceEditingPlaceholder>
+        Click to select posted at.
       </InPlaceEditingPlaceholder>
     );
   }
 
-  return <span>{formatDate(datePosted, "mm/dd/yyyy")}</span>;
-});
+  return formatDate(datePosted, "mm/dd/yyyy");
+}
+
+const JobValidThrough = Scrivito.connect(({ page }) => {
+  const validThrough = page.get("validThrough");
+  if (!validThrough) {
+    if (!Scrivito.isInPlaceEditingActive()) {
+      return null;
+    }
+
+    return (
+      <h2 className="h5">
+        <i className="fa fa-calendar-o fa-lg" aria-hidden="true" title="date" />{" "}
+        <span className="font-weight-bold">Valid through: </span>
+        <Scrivito.ContentTag content={page} attribute="validThrough" tag="span">
+          <InPlaceEditingPlaceholder>
+            Click to select expire at.
```

Comment:
* [ ] Maybe simply `Click to select a date`, or `Click to set an expiry date`. (Also, the label already gives some context: "valid through".)